### ISSUE TITLE
Duplicate cryptography dependency with different versions

### ISSUE THREAD

**Issue by sdispater** [MasoniteFramework/core#136]
When I was testing the new dependency resolver for poetry, I saw that there were conflicts when executing:

```
poetry debug:resolve masonite -vv
```

which led to masonite 1.5.7 being selected rather than the latest one (1.6.7)

It appears that cryptography is mentioned twice in the dependencies: [here](https://github.com/MasoniteFramework/core/blob/1.6/setup.py#L21) and [here](https://github.com/MasoniteFramework/core/blob/1.6/setup.py#L32)

This is made clearer by executing:

```
poetry debug:resolve masonite=1.6.7
```

which gives:

```
[SolverProblemError]
Because masonite (1.6.7) depends on both cryptography (2.1.4) and cryptography (2.2.2),
 masonite is forbidden.
So, because demo depends on masonite (1.6.7), version solving failed.
```

**Comment by josephmancuso** [comment]
Fixed in 1.6.8 from 08003d29d58e9441e7c6357022b2f2a494969907

**Comment by josephmancuso** [comment]
and that's actually funny because pipenv didn't pick them up
